README.md

```diff
@@ -32,10 +115,242 @@ Example
+### Server Setup
+
+The server will consist of docker containers such as a staging and production API, PostgreSQL database and a Minio database. It will integrate with Github actions to streamline development and automaticly update Staging and Production API to follow newest releases.
+To set this up correctly please follow these steps.
+
+1. First step is to figure out where to host the server. A great place is AAU's own hosting platform https://strato-new.claaudia.aau.dk here the most important thing is to pick a server running Ubuntu.
+
+> [!IMPORTANT]
+> If Strato is decided then remember to open all the ports that are expected to be used. This can be done in Security Groups under the Network section. These ports could be 5432, 8080, 8081, 9000 and 9001
+
+2. Install the following on the server:
+
+-   [Docker](https://docs.docker.com/engine/install/ubuntu/)
+-   [Cron](https://www.digitalocean.com/community/tutorials/how-to-use-cron-to-automate-tasks-ubuntu-1804)
+
+3. Create a new file called `docker-compose.yml` and open it using the following command
+
+    ```bash
+    touch docker-compose.yml
+    nano docker-compose.yml
+    ```
+
+    Paste the following code into the file. **Remember to update the variables with your own information.**
+
+    ```yml
+    version: "3.8"
+
+    services:
+        minio:
+            image: minio/minio:latest
+            container_name: minio_giraf
+            restart: unless-stopped
+            ports:
+                - "9000:9000"
+                - "9001:9001"
+            volumes:
+                - ./minio/data:/mnt/data
+            environment:
+                - MINIO_ROOT_USER=<insert here>
+                - MINIO_ROOT_PASSWORD=<insert here>
+                - MINIO_VOLUMES=/mnt/data
+            command: server /mnt/data --console-address ":9001"
+
+        postgres:
+            image: postgres:latest
+            container_name: postgres_giraf
+            restart: unless-stopped
+            ports:
+                - "5432:5432"
+            volumes:
+                - ./postgres/data:/var/lib/postgresql/data
+            environment:
+                - POSTGRES_PASSWORD=<insert here>
+                - POSTGRES_USER=<insert here>
+                - POSTGRES_DB=<insert here>
+    ```
+
+    Go ahead and run the docker compose file using the following command:
+
+    ```bash
+    docker compose -f docker-compose.yml up
+    ```
+
+    This will create two containers containing **Minio** for image storage and PostgreSQL for data storage.
+    The Minio server can now be accessed and managed on `http://<server-ip>:9001`
+
+4. Connect to the newly created PostgreSQL container using your prefered PostgreSQL database tool ex. [pgAdmin](https://www.pgadmin.org/download/).
+
+> [!TIP]
+> **Host name/address**: This is your server-ip<br>
```

Comment:
Evt. tilføje example for localhost, så det er ez at forstå

Reply:
Det sted i koden handler jo om server opsætning, så ved ikke helt om det vil passe med localhost eller?